.github/workflows/continous-integration-docker.yml

```
@@ -8,6 +8,12 @@ concurrency:
   cancel-in-progress: true
 
 on:
+  schedule:
+    # Run this job every Friday at mid-day UTC
+    # This is limited to the Zebra Full Sync job
```

Can you add the full list of jobs here?

```suggestion
    # This is limited to the Zebra and lightwalletd Full Sync jobs
```